apache_httpd-formula/tests/reference/etc/apache2/vhosts.d/mysite2.conf

```diff
@@ -0,0 +1,15 @@
+# Managed by the apache_httpd formula
+
+<VirtualHost *:80>
+    ServerName mysite2.example.com
+    CustomLog /var/log/apache2/mysite2-access.log combined
+    ErrorLog /var/log/apache2/mysite2-error.log
+    ServerName mysite2.example.com
```

Comment:
This config contains `ServerName mysite2.example.com` twice.

Reply:
Interesting find.

Reply:
[dd9d3fb](https://github.com/openSUSE/salt-formulas/pull/148/commits/dd9d3fb2c20ba85b1c5ee7d2f0a28cd64fb4d463)